src/views/Simulation/components/Charts/TimelineControls.js

```diff
@@ -94,9 +96,33 @@ const TimelineControls = ({ chartData, markers, startDate, endDate }) => {
   const animationFrameId = useRef(null);
   const startTimeRef = useRef(0);
   const pausedAtRef = useRef(0);
-  const isPlayingRef = useRef(false); // live flag
+  const isPlayingRef = useRef(false);
+
+  const stepDurationRef = useRef(500);
```

Comment:
`stepDurationRef` value is never changed, maybe we could move it in a `const` above the component?
(e.g. define `const STEP_DURATION_IN_MS = 500;` just after `SPEEDS` definition)